db/migrate/20240426135118_add_pack_models.rb

```diff
@@ -0,0 +1,21 @@
+class AddPackModels < ActiveRecord::Migration[7.0]
+  def change
+    create_table :request_units do |t|
+      t.string :name, null: false
+      t.references :organization, foreign_key: true
+      t.timestamps
+    end
+
+    create_table :item_request_units do |t|
+      t.string :name, null: false
+      t.references :item, foreign_key: true
+      t.timestamps
+    end
+
+    safety_assured do
+      add_column :organizations, :uses_request_units, :boolean, null: false, default: false
+      add_column :items, :reporting_unit, :string
+    end
```

Comment:
As commented elsewhere, I think we can get rid of the reporting unit on items.   

I also wonder if we really need the explicit uses_request_units on Organization or if that would be served as well with a wee routine that checks whether the Organization has any request units.    We aren't explicitly asking the users if they use custom request units, but just have them put them in if we are.  Thoughts?

Reply:
I'm good with that!